src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/ConnectionDetails.cs

```diff
@@ -99,6 +99,22 @@ public string AuthenticationType
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value that specifies that Always Encrypted functionality is enabled in a connection.
+        /// </summary>
+        public string ColumnEncryptionSetting
```

Comment:
Shouldn't this be be Boolean line "Encrypt"?

Reply:
I thought so at first but it actually gets sent over from ADS in the JSON RPC request as a string value.

Reply:
it's because how it's configured here:  [https://github.com/microsoft/sqltoolsservice/blob/master/src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionProviderOptionsHelper.cs](https://github.com/microsoft/sqltoolsservice/blob/master/src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionProviderOptionsHelper.cs)

---
In reply to: [366025319](https://github.com/microsoft/sqltoolsservice/pull/908#discussion_r366025319) [](ancestors = 366025319)

Reply:
we can keep it as is since smo also gets the value of this as "enabled"/"disabled" not boolean

---
In reply to: [366494485](https://github.com/microsoft/sqltoolsservice/pull/908#discussion_r366494485) [](ancestors = 366494485,366025319)